<?xml version="1.0" ?>
<sdf version='1.4'>
  <model name="yellow_cube">      
      <static>0</static>
      <link name='link'>
        <inertial>
          <mass>0.2</mass>
          <inertia>
            <ixx>0.15</ixx>
            <ixy>0.000000</ixy>
            <ixz>0.000000</ixz>
            <iyy>0.15</iyy>
            <iyz>0.000000</iyz>
            <izz>0.15</izz>
          </inertia>
        </inertial>
        <collision name='collision'>
          <geometry>
            <box>
              <size>0.35 0.35 0.35</size>
            </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <friction>
              <ode>
                <mu>0.03</mu>
                <mu2>0.03</mu2>
              </ode>
            </friction>
            <contact>
              <ode/>
            </contact>
            <bounce/>
          </surface>
        </collision>
        <visual name='visual'>
          <geometry>
            <box>
              <size>0.35 0.35 0.35</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Yellow</name>
            </script>
          </material>
        </visual>
        <velocity_decay>
          <linear>0.000000</linear>
          <angular>0.000000</angular>
        </velocity_decay>
        <self_collide>0</self_collide>
        <kinematic>0</kinematic>
        <gravity>1</gravity>        
      </link>   
  </model>
</sdf>
